src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/MLModel.scala

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.partnercatalyst.fortis.spark.dto
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+import java.util.Date
+
+object MLModel {
+
+  def dataFrom(modelObj: Serializable): Array[Byte] = {
+    val stream = new ByteArrayOutputStream()
+    val oos = new ObjectOutputStream(stream)
+    oos.writeObject(modelObj)
```

Comment:
Apache-Commons has a nice [helper](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SerializationUtils.html#serialize-java.io.Serializable-) for this serialization and deserialization:

```scala
import org.apache.commons.lang.SerializationUtils.serialize;
return serialize(modelObj);
```

In case you want to use this:

```scala
// https://mvnrepository.com/artifact/commons-lang/commons-lang
libraryDependencies += "commons-lang" % "commons-lang" % "2.6"
```